src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  outline: none;
+  height: 100%;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+  height: 70px;
+  background: #fff;
+}
+
+.form-small {
+  height: 42px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+  width: 19px;
+  height: 19px;
+  padding: 0;
+  background-image: url(./images/Search.svg);
+  background-color: #fff;
+  background-repeat: no-repeat;
+  border: none;
+  color: transparent;
+  cursor: pointer;
+}
+
+.icon-small {
+  top: 15px;
+  left: 13px;
+  bottom: 16px;
+  width: 11px;
+  height: 11px;
+  background-size: 11px 11px;
+}
+
+.field {
+  width: 100%;
+  padding: 23px 0 23px 62px;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 21.86px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.field-small {
+  padding: 10px 0 10px 33px;
+  line-height: 19.12px;
+  font-size: 14px;
+}
+
+.field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.field::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+  line-height: 21.86px;
+}
+
+.field-small::placeholder {
+  font-size: 14px;
```

Comment:
You don't need separate font-size for placeholder. It should inherit input's font-size. And in your case it is also 14px. If font-size of placeholder is not equal to input's font-size then check carefully your styles, it means that you have overwritten something somewhere